src/firefly/java/edu/caltech/ipac/firefly/server/util/VersionUtil.java

```diff
@@ -120,6 +123,24 @@ private static String getFireflyUA() {
             return unknownVersionUA;
         }
     }
+
+    public static String getVersionInfo() {
+        Version v = getAppVersion();
+
+        List<String> infoStrings = new ArrayList<>(Arrays.asList(
+                "Version: " + v.getBuildTag(), // TODO: Replace by version parsing (from VersionInfo.jsx)
+                "Built On: " + v.getBuildTime(),
+                "Git commit: " + v.getBuildCommit()));
```

Comment:
This code works fine.  Maybe remove the commented out lines, since I don't see any value in keeping them.

However, since you are new to Java server-side programming, I will give you this feedback.
`new ArrayList<>(Arrays.asList(` is very inefficient use of memory, since you're creating 2 Lists just because Arrays.asList is immutable.  

If you want to use a list, you can just create one, then add these 3 items to it.
A better approach would be to use a StringBuilder, e.g.  
```.java
        StringBuilder vInfo = new StringBuilder();
        vInfo.append("\nVersion: " + v.getBuildTag());
        vInfo.append("\nBuilt On: " + v.getBuildTime());
        vInfo.append("\nGit commit: " + v.getBuildCommit());
        return vInfo.toString();
```

Reply:
So I realized we need differently formatted strings for console log and /admin/status panel. So I created this as a list of KeyVal. Please feel free to let me know if what I have done now, can be further improved in efficiency?